doc/misc/npm-scripts.md

```diff
@@ -43,7 +48,30 @@ names will be run for those as well (e.g. `premyscript`, `myscript`,
 `postmyscript`). Scripts from dependencies can be run with `npm explore
 <pkg> -- npm run <stage>`.
 
-## COMMON USES
+## PREPUBLISH AND PREPARE
+
+### DEPRECATION NOTE
+
+Since `npm@1.1.71`, the npm CLI has run the `prepublish` script for both `npm
+publish` and `npm install`, because it's a convenient way to prepare a package
+for use (some common use cases are described in the section below).  It has
+also turned out to be, in practice, [very
+confusing](https://github.com/npm/npm/issues/10074).  As of `npm@4.0.0`, a new
```

Comment:
It is a constant struggle. I love my team, but they still double-space after their periods, in an age of kerning. 😭